src/apps/vpn/serverlatency.cpp

```diff
@@ -29,6 +29,9 @@ constexpr const int SERVER_LATENCY_MAX_RETRIES = 2;
 // Delay the progressChanged() signal to rate-limit how often score changes.
 constexpr const uint32_t SERVER_LATENCY_PROGRESS_DELAY_MSEC = 500;
 
+// Minimum number of redundant servers we expect at a location.
+constexpr int SCORE_SERVER_REDUNDANCY_THRESHOLD = 3;
```

Comment:
Random thought for the future. It would be fun to think about making this configurable by an add-on so we can then run experiments on what a good threshold is.